libtelemetry/src/test/java/com/mapbox/services/android/telemetry/MapEventFactoryTest.java

```diff
@@ -29,7 +29,7 @@ public void checksMapLoadEvent() throws Exception {
     initializeMapboxTelemetry();
     MapEventFactory aMapEventFactory = new MapEventFactory();
 
-    Event mapLoadEvent = aMapEventFactory.createMapLoadEvent(Event.Type.MAP_LOAD);
+    Event mapLoadEvent = aMapEventFactory.createMapLoadEvent(Event.Type.MAP_LOAD, "aUserId");
```

Comment:
This is actually a _valid_ user id.
The same thing happens in the tests below 👇
We should add the missing _not valid_ tests.